src/views/home/index.js

```diff
@@ -12,7 +12,7 @@ function Home({sessions}) {
             {/* TODO: HOME VIEW */}
             <Header/>
             <Menu/>
-            <Gallery sessions={sessions}/>   
+            <Thumbnail sessions={sessions}/>   
```

Comment:
This doesn't seem right, a Thumbnail component but actually a gallery/collection of Thumbnails...